Makefile

```diff
@@ -57,7 +57,7 @@ help:
 	@echo 'Examples:'
 	@echo
 	@echo '  # Example 1: set up personal development environment'
-	@echo '  $$ GITHUB_USER=johndoe MINIKUBE_DRIVER=virtualbox make setup prefetch clone'
+	@echo '  $$ GITHUB_USER=johndoe make setup prefetch clone'
```

Comment:
Since you are changing this line, can you reorder the steps? It should be `make setup clone prefetch` (`prefetch` uses `reana-dev` and `clone` installs it) :)

Reply:
Updated.